components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -122,6 +122,18 @@ export class TeamDBImpl implements TeamDB {
         return soleOwnedTeams;
     }
 
+    public async updateTeam(team: Partial<Team>): Promise<Team> {
```

Comment:
The signature here is possibly misleading. You can only update the team name, but you can pass in the whole team. Does it make sense to be explicit with which fields are acceptable?

Reply:
Do we need to also check the team name is not empty? Currently I could set the name to empty